meta-resin-common/recipes-support/hostapp-update-hooks/files/0-bootfiles

```diff
@@ -53,35 +52,50 @@ isModified() {
 	fi
 }
 
-# Copies a file from a container to the boot filesystem atomically and durable
+# Copies a file from /resin-boot to boot partition filesystem atomically and durable
 # Arguments:
 # 	$1: boot partition file
-copyFromContainer() {
+copyBootFile() {
 	local _file="$1"
 	mkdir -p "$boot_mountpoint/$(dirname "$_file")"
-	docker cp "$container_id:/resin-boot/$_file" "$boot_mountpoint/$_file.new"
-	sync -f $boot_mountpoint
-	mv "$boot_mountpoint/$_file.new" "$boot_mountpoint/$_file"
-	sync -f $boot_mountpoint
+	if cp "/resin-boot/$_file" "$boot_mountpoint/$_file.new"; then
+		sync -f $boot_mountpoint
+		mv "$boot_mountpoint/$_file.new" "$boot_mountpoint/$_file"
+		sync -f $boot_mountpoint
+	else
+		# Cleanup all new files we deployed
+		find "$boot_mountpoint" -type f -name "*.new" -exec rm -f {} \;
+		for file in $new_deployed_files; do
+			rm -f "$boot_mountpoint/$file"
+		done
+		exit 1
+	fi
 }
 
-# Deploys file from docker container to boot partition
+# Deploys files to boot partition
 # Arguments:
 # 		$1: file path relative to boot partition's root
 deploy() {
 	local _file="$1"
 	printf "[INFO] Deploying %s%s..." "${boot_mountpoint}" "${_file}"
-	if isModified "$_file"; then
-		if isBlacklisted "$_file"; then
-			printf " file modified but blacklisted. Ignoring.\n"
+	if isBlacklisted "$_file"; then
+		printf " file modified but blacklisted. Ignoring.\n"
```

Comment:
The code before seemed to check if the file is both modified and blacklisted but now it only checks if it's blacklisted. We should change the message

Reply:
Done. Will push with next iteration.